back/engines/commercial/report_builder/app/policies/report_builder/live_graph_data_unit_policy.rb

```diff
@@ -10,7 +10,9 @@ def live?
         project = Project.find(props[:project_id])
         policy_for(project).active_moderator?
       else
-        active_admin?
+        # For global project widgets (like ProjectsTimelineWidget, ProjectsWidget),
+        # allow access if user is admin or project moderator
+        active_admin? || user&.project_moderator? || user&.project_folder_moderator?
```

Comment:
We cannot just open this up like this. This permission system works like this for a reason, if you add this logic then all the lines above are basically pointless. These lines are there because we can't just give project moderators access to all this data related to projects that they can't moderate.